mod/file/actions/file/upload.php

```diff
@@ -108,61 +104,76 @@
 
 	$guid = $file->save();
 
-	// if image, we need to create thumbnails (this should be moved into a function)
-	if ($guid && $file->simpletype == "image") {
-		$file->icontime = time();
-		
-		$thumbnail = get_resized_image_from_existing_file($file->getFilenameOnFilestore(), 60, 60, true);
-		if ($thumbnail) {
-			$thumb = new ElggFile();
```

Comment:
We didn't set the owner so we were assuming the logged in user owned it. Was this another bug?